duckduckgo/Android
# No engagement notifications experiment

Task/Issue URL: https://app.asana.com/0/488551667048375/1205564862194177/f

### Description

- Disabled set as default experiment
- Implemented new retention experiment where we won't show engagement notifications for the experimental group

### Steps to test this PR

_Change all variants to 0 except for the `md` variant_
_Change delay time to schedule engagement notifications in `AndroidNotificationScheduler.kt` (Lines 62 & 70) so you can see notifications after seconds instead days_

- Uninstall previous version of the app or clear storage
- Install from this branch
- [x] Check you don't see engagement notifications after the delay time

### No UI changes
